frame/support/src/traits/tokens/currency.rs

```diff
@@ -176,7 +176,7 @@ pub trait Currency<AccountId> {
 	}
 
 	/// Removes some free balance from `who` account for `reason` if possible. If `liveness` is
-	/// `KeepAlive`, then no less than `ExistentialDeposit` must be left remaining.
+	/// `Expendability`, then no less than `ExistentialDeposit` must be left remaining.
```

Comment:
it says:
> if `liveness` is `Expendability`
but `liveness` (`ExistenceRequirement`) can only be either `KeepAlive` or `AllowDeath`
```suggestion
	/// `KeepAlive`, then no less than `ExistentialDeposit` must be left remaining.
```